dockers/docker-lldp/lldpmgrd

```diff
@@ -50,23 +50,53 @@ class LldpManager(daemon_base.DaemonBase):
         # Open a handle to the Config database
         self.config_db = swsscommon.DBConnector("CONFIG_DB",
                                                 self.REDIS_TIMEOUT_MS,
-                                                True)
+                                                False)
 
         # Open a handle to the Application database
         self.appl_db = swsscommon.DBConnector("APPL_DB",
                                               self.REDIS_TIMEOUT_MS,
-                                              True)
+                                              False)
 
         self.pending_cmds = {}
+        self.hostname = "None"
+        self.mgmt_ip = "None"
+
+        self.device_table = swsscommon.Table(self.config_db, swsscommon.CFG_DEVICE_METADATA_TABLE_NAME)
+        self.port_table = swsscommon.Table(self.config_db, swsscommon.CFG_PORT_TABLE_NAME)
+        self.mgmt_table = swsscommon.Table(self.config_db, swsscommon.CFG_MGMT_INTERFACE_TABLE_NAME)
+        self.app_port_table = swsscommon.Table(self.appl_db, swsscommon.APP_PORT_TABLE_NAME)
+
+    def update_hostname(self, hostname):
+        cmd = "lldpcli configure system hostname {0}".format(hostname)
+        self.log_debug("Running command: '{}'".format(cmd))
```

Comment:
can we this function generic so that it be extendable future
{ update_type: [set cmd, uset cmd] }

Reply:
I dint get this comment, can you please provide more info or share an existing code reference if there is one available. Also in CLICK there is no option to unconfigure the hostname, we have to configure the default name back for "no" equivalent of the command.

Reply:
@sandeep-kulambi I am thinking of one API to execute any command. Command can present present in dictionary with key as operation type. Basically both update function can be collapse into one generic function ?

Reply:
@abdosi - Thanks for clarifying, the current approach is inline with legacy code, changing it has the implication that we will have 2 different implementations. Also with dictionary approach we might make the code less readable, since we need to take care of configure/unconfigure/different inputs, also only if command is successful we need to update the corresponding information that is locally maintained, all this would bring in additional logic. Considering this will prefer to not change this.